modules/mobile/app/controllers/mobile/v0/payment_information_controller.rb

```diff
@@ -23,6 +23,7 @@ class PaymentInformationController < ApplicationController
       def index
         payment_information = if lighthouse?
                                 data = lighthouse_service.get_payment_info
+                                validate_response!(data)
```

Comment:
I'm writing this code in the controller instead of in the adapter in order to keep it a little more visible. Hopefully this code won't live long.